tests/Data/ProgrammesDb/EntityRepository/PromotionRepository/FindActiveSuperpromotionsByPidTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+
+use DateTimeImmutable;
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\PromotionRepository::<public>
+ */
+class FindActiveSuperpromotionsByPidTest extends AbstractDatabaseTest
+{
+    private $promotionRepository;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->loadFixtures(['PromotionsFixture']);
+        $this->promotionRepository = $this->getRepository('ProgrammesPagesService:Promotion');
+    }
+
+    public function testActiveSuperpromotionsAreReceivedACoreEntity()
+    {
+        $coreEntityRepo = $this->getEntityManager()->getRepository('ProgrammesPagesService:CoreEntity');
+
+        $series = $coreEntityRepo->findByPid('b00swyx1', 'Series');
+        $serieDbAncestryIds = explode(',', $series['ancestry']);
```

Comment:
`$seriesDbAncestryIds` instead of `$serieDbAncestryIds`. You still have serie in several places.